continuous_integration/azure/runtests.bat

```diff
@@ -0,0 +1,8 @@
+python setup.py sdist
```

Comment:
This should be

    python setup.py develop

because `sdist` builds a `tar.gz` file with the contents of this package, but we don't need that for testing.